### DISCUSSION TITLE
Logo proposal

### DISCUSSION THREAD

**Discussion by lincolnbrito** [wintercms/winter#72] [Ideas]
I'm very excited with the project and I would like to contribute with visual identity

Here are some logo proposals:

![winter-logos](https://user-images.githubusercontent.com/310896/114639760-00aff300-9ca5-11eb-974a-839b3a112282.png)

**Comment by LukeTowers** [comment]
I do like these, but what's wrong with the current logo? 😉

**Reply by seanthepottingshed** [reply]
@LukeTowers personally I think the current logo is a bit too Christmas rather than CMS like

**Reply by Awilum** [reply]
Agree

**Reply by Awilum** [reply]
Logo should be simple and memorable. Current Winter CMS logo isn't simple.

**Reply by bennothommo** [reply]
I'd argue that these are more "complex" than our current logo, TBH. (not saying they're bad though - although not the biggest fan of 1).

**Comment by pablorq** [comment]
N3

**Reply by HitArrowLegend** [reply]
N3 every day

**Comment by Pok09** [comment]
N° 01

**Comment by OVillani** [comment]
Just an idea to emphasis the community approach ;)
![WCMS1](https://user-images.githubusercontent.com/6983432/115569838-e2a64c00-a2bd-11eb-83ed-d30495013a9d.jpg)

**Reply by rubenvanerk** [reply]
Creative!

**Reply by bennothommo** [reply]
@ovillani Interesting idea - but not sure of the colours. Here's our branding guide - did you want to give it an attempt implementing these colours?

![Winter CMS Branding](https://user-images.githubusercontent.com/15900351/115587326-23bf5000-a300-11eb-9f2b-d1198d2bfa38.png)

**Reply by OVillani** [reply]
yes, sure I do. Will have some time left this evening.

**Reply by LukeTowers** [reply]
Getting some major Joomla vibes from this one 😂 Unlikely that we'll change our main project logo at the moment, but I like seeing what people come up with, and we can always look at using some of these for plugins or community projects.

**Comment by OVillani** [comment]
So here are two approaches with CD colors, one more naturalistic, one more abstract.
![WCMS-logo-natural-sml](https://user-images.githubusercontent.com/6983432/115616624-b22bd580-a2f0-11eb-993a-0ff87a68bcc7.jpg)

...

![WCMS-logo-abstract-sml](https://user-images.githubusercontent.com/6983432/115616911-0cc53180-a2f1-11eb-85e3-462e227bf3e4.jpg)

**Reply by LukeTowers** [reply]
How would it look with the tops of the arms slanted so they maintain the same line from the heads?

**Reply by OVillani** [reply]
Hm, good idea!

**Reply by OVillani** [reply]
Did you mean it like the a) or b) version or something else?
![WCMS-logo-natural rev01jpg](https://user-images.githubusercontent.com/6983432/115619842-b35f0180-a2f4-11eb-9241-7b9a6f1bf8e7.jpg)

**Reply by LukeTowers** [reply]
Closer to a, but leave the arms the same width throughout as well as keep the heads triangles. Just change the cap on the arms to an angle to match up with the line of the head, to create the visual idea of a hexagon wrapping the logo.

**Reply by OVillani** [reply]
Just another scribble for brainstorming :) 

![WCMS-logo-natural rev02](https://user-images.githubusercontent.com/6983432/115624238-9fb69980-a2fa-11eb-8c74-36f129dae61a.jpg)

**Reply by OVillani** [reply]
> 
> 
> Closer to a, but leave the arms the same width throughout as well as keep the heads triangles. Just change the cap on the arms to an angle to match up with the line of the head, to create the visual idea of a hexagon wrapping the logo.

Think I got it. Will give it a try :)

**Reply by OVillani** [reply]
Sooooo....

![WCMS-logo-rev03](https://user-images.githubusercontent.com/6983432/115627388-3422fb00-a2ff-11eb-8e85-dabcb7343384.jpg)

**Reply by OVillani** [reply]
Honestly, not that happy. Looks somewhat like the "Reichsadler" :( or is this just to me?

**Reply by OVillani** [reply]
... and the snowflake impression is not so strong anymore...

**Reply by OVillani** [reply]
rev.4: Shortening the arms. No more Reichsadler (as in rev.3) instead more "snowflakeish" but hexagon approach better than in rev.2 ...

![WCMS-logo-rev04](https://user-images.githubusercontent.com/6983432/115629065-e78cef00-a301-11eb-9002-93f5cf6cb4fc.jpg)

**Reply by pablorq** [reply]
Nice ideas, but as @bennothommo said, it seems to be "complex".

I remain with number 3, which is a simplification and stronger version of the current logo.

**Reply by OVillani** [reply]
:)

**Reply by sng82** [reply]
Could you use the 'W' in the snowflake? I'm no designer, more of an _MS Paint Hero_ so feel free to ignore me!
![winter](https://user-images.githubusercontent.com/18116407/115695333-9c9ec600-a359-11eb-92bc-e946e6b7f522.png)

**Reply by HitArrowLegend** [reply]
MS Paint is a very reputable software among the graphic designer community 🕺🕺🕺

**Reply by OVillani** [reply]
> 
> 
> MS Paint is a very reputable software among the graphic designer community 🕺🕺🕺

Sure on that, but it's more the idea that counts. And the W idea is not bad too.

"The best way to a good idea is to have a lot of ideas" Linus Pauling :)

**Reply by sng82** [reply]
> Sure on that, but it's more the idea that counts. And the W idea is not bad too.

Yupp, the image was just to illustrate the idea. Of course, you're welcome to use my janky logo as-is, but I'll think less of you if you do! ;)

**Reply by fosterushka** [reply]
> Could you use the 'W' in the snowflake? I'm no designer, more of an _MS Paint Hero_ so feel free to ignore me!
> ![winter](https://user-images.githubusercontent.com/18116407/115695333-9c9ec600-a359-11eb-92bc-e946e6b7f522.png)

i like this doe

**Reply by LukeTowers** [reply]
Reminds me of [Magneton](https://bulbapedia.bulbagarden.net/wiki/Magneton_(Pok%C3%A9mon)) from Pokèmon 😂

**Reply by Awilum** [reply]
@ovillani it is funny logo :D

**Comment by drmzio** [comment]
I like the 3rd one a lot!

Current logo is great too, but it isn't very scalable in small variants (see the favicon)

**Comment by willnv** [comment]
My thoughts:
Current logo looks bad IMO, has no personality, looks like a random snowflake taken out of google images.

@ovillani's choice doesn't work on white backgrounds, lower-left arm is too close to white. 
The idea of a colored logo reminds me of a children support org or something... Too many distant colors don't inspire security, modernity, simplicity, etc...

**Edit:** there are other things that can work as a symbol to winter other than the obvious snow flake e.g. polar bear, snowy mountain, winter wolf

**Reply by OVillani** [reply]
Hi,
I don't see the current logo that bad :)

You are right about the colour of the lower-left arm. It is one of the branding colours - I'll give it a try with a slightly darker version. But please consider, my suggestions are only a brainstorming!

**Comment by bennothommo** [comment]
Thanks everyone for your thoughts on this.

Logos are unfortunately one of those things where it's highly subjective - even in this thread, there's some that like the current logo and others that don't. There's also a wide range of feedback on the logos provided by @ovillani, @sng82, among others.

Since we've only just started using our current logo, we're gonna give it a little bit of a run first before switching, but we've taken all your feedback on board and will definitely consider our options at some point.

**Reply by OVillani** [reply]
The best way to get a good idea is to have a lot of ideas :)

**Reply by drmzio** [reply]
Agreed. Logo should not be a priority right now, only until Winter CMS and its ecosystem is settled 👍
